plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/Module.kt

```diff
@@ -21,6 +22,8 @@ internal sealed class Module {
             get() = extension.libraryVariants.find {
                 it.name.contains(variantName, ignoreCase = true)
```

Comment:
How will adding multiple variant specific tasks affect using the specified `variantName` here?

Reply:
See latest update. I've removed that property now, incorporating the functionality into the new changes.